cmd/util/cmd/execution-state-extract/execution_state_extract.go

```diff
@@ -78,19 +77,7 @@ func extractExecutionState(
 	newState := ledger.State(targetHash)
 
 	if migrate {
-		storageUsedUpdateMigration := mgr.StorageUsedUpdateMigration{
-			Log:       log,
-			OutputDir: outputDir,
-		}
-		accountStatusMigration := mgr.NewAccountStatusMigration(log)
-		legacyControllerMigration := mgr.LegacyControllerMigration{Logger: log}
-
-		migrations = []ledger.Migration{
-			accountStatusMigration.Migrate,
-			legacyControllerMigration.Migrate,
-			storageUsedUpdateMigration.Migrate,
-			mgr.PruneMigration,
```

Comment:
We don't need these two if there is no other migration applied

Reply:
Do we no longer need the `PruneMigration` If I remember correctly we always ran that.